rules/common/commentDocPreviewLink.ts

```diff
@@ -0,0 +1,10 @@
+import { danger, warn } from "danger"
+
+// Comment the Deploy Preview Link
+const commentDocPreviewLink = () => {
+  const repo = danger.github.repository.name
+  const branch = danger.github.pr.head.ref
+  warn(`Deploy preview available at https://docs.loadsmart.io/${repo}/${branch}/index.html`)
```

Comment:
I believe `message` instead of `warn` would be more appropriate here, @camilamaia.

Reply:
@barbosa indeed. Fixed!